src/main/resources/site/content-types/highchart/highchart.xml

```diff
@@ -51,6 +51,14 @@
       <help-text>Grupperte verdier kan vises hver for seg (ingen stabling) eller summeres i en felles søyle med stabling</help-text>
     </input>
 
+    <input name="showStackedTotal" type="Checkbox">
+      <label>Vis stabelsum</label>
```

Comment:
Fra jirasaken: Endre feltets tittel fra "Vis stablesum" til "Vis samlet sum for stolpe".